keyboards/pabile/p18/config.h

```diff
@@ -0,0 +1,27 @@
+#pragma once
+
+#include "config_common.h"
+
+/* USB Device descriptor parameter */
+#define VENDOR_ID       0x6666
```

Comment:
I'm not a huge fan of using this VID, but ... I suppose it should be okay.

Reply:
i tried to do some reading found this as "available" or something that i can use. i tried to follow this VID with my other keyboards and gut stuck.